src/covid_model_seiir_pipeline/forecasting/task/beta_forecast.py

```diff
@@ -106,7 +106,7 @@ def run_beta_forecast(draw_id: int, forecast_version: str, scenario_name: str):
                     .sort_values(['location_id', 'date'])
                     .set_index(['location_id']))
     components['theta'] = thetas.reindex(components.index).fillna(0)
-    data_interface.save_components(components, scenario_name, draw_id)
+    data_interface.save_components(components.reset_index(), scenario_name, draw_id)
```

Comment:
I kind of wish `components = components.reset_index()  # restore location_id column prior to save` was it's own line

Reply:
The thing is, I don't want to reset the index for the computations below, just for saving.

Reply:
Maybe add a comment then? I find it very unintuitive that you're serializing something that is **not quite** what you're using to process data.

Reply:
Fair.  Will add some comments in the next PR.  I need to rework this section to allow for multiple algorithms anyway.